File: android/src/main/java/me/carda/awesome_notifications/awesome_notifications_core/listeners/AwesomeLifeCycleEventListener.java
package me.carda.awesome_notifications.awesome_notifications_core.listeners;

import android.app.Activity;

import androidx.lifecycle.Lifecycle;

import me.carda.awesome_notifications.awesome_notifications_core.enumerators.NotificationLifeCycle;

public interface AwesomeLifeCycleEventListener {
    public void onNewLifeCycleEvent(NotificationLifeCycle lifeCycle);
}

File: reference/awesome_notifications/ios/AwnCore/Classes/listeners/AwesomeLifeCycleEventListener.swift
//
//  AwesomeLifeCycleEventListener.swift
//  awesome_notifications
//
//  Created by CardaDev on 30/01/22.
//

import Foundation

public protocol AwesomeLifeCycleEventListener: AnyObject {
    func onNewLifeCycleEvent(lifeCycle:NotificationLifeCycle)
}
